app/components/settings/api-tokens.hbs

```diff
@@ -97,9 +97,9 @@
           its value again. For use on the command line you can save it to <code>~/.cargo/credentials</code>
           with:
 
-          <pre class="terminal">cargo login {{token.token}}</pre>
+          <pre class="terminal"> cargo login {{token.token}}</pre>
```

Comment:
I'm wondering if we should suggest this command at all. Maybe we should only print the token itself and suggest `cargo login` usage without the token argument in the surrounding text. 🤔

Reply:
I'm on board with that idea!